zingo-testutils/src/lib.rs

```diff
@@ -243,6 +244,272 @@ pub async fn load_wallet(
     )
 }
 
+#[derive(Debug)]
+pub struct TxNotes {
```

Comment:
maybe this should be called TxNotesCount?

Reply:
change made in [27144bd]